roles/swift-ring/handlers/main.yml

```diff
@@ -1,5 +1,19 @@
 ---
-- name: setup swift rings
+- name: setup swift rings - bootstrap
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
-           bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
+           --config {{ swift_ring.outdir }}/ring_definition.yml --outdir {{ swift_ring.outdir }} bootstrap
```

Comment:
Instead of `sudo -u swiftops` in the command, please use
```
become: true
become_user: swiftops
```
as task arguments. This confirms with Ansible 2.x conventions.